models/stg_stripe__invoice.sql

```diff
@@ -43,8 +43,9 @@ final as (
         tax,
         tax_percent,
         total,
-        subscription_id
-        
+        subscription_id,
+        period_start,
+        period_end
```

Comment:
Would you be able to add these fields and descriptions to the invoice tables in the `stg_stripe.yml` and `src_stripe.yml` files.

Reply:
Added to both the files